apps/finder.py

```diff
@@ -0,0 +1,34 @@
+from talon.voice import Key, Context, Str, press
+
+def go_to_path(path):
+	def path_function(m):
```

Comment:
could you please convert tabs into spaces

Reply:
It is weird, seems like pylint is not working on my machine and that makes it impossible to change to spaces, it changes back to tabs every time I close the file :/

Reply:
got it...